kuma/contributions/forms.py

```diff
@@ -118,12 +118,22 @@ def make_charge(self):
             amount = amount * 100
         token = self.cleaned_data.get('stripe_token', '')
         if token and amount:
-            # TODO: Handle API errors
-            # See https://stripe.com/docs/api/python, "Handling Errors"
-            charge = stripe.Charge.create(
-                amount=amount,
-                currency='usd',
-                source=token,
-                description="Contribute to MDN Web Docs"
-            )
-        return charge
+            try:
+                stripe.Charge.create(
+                    amount=amount,
+                    currency='usd',
+                    source=token,
+                    description='Contribute to MDN Web Docs',
+                    receipt_email=self.cleaned_data['email'],
+                    metadata={'name': self.cleaned_data['name']}
+                )
+                return True
+            except Exception as e:
```

Comment:
I'm not a fan of generic exception handlers, but this worked for the card error for ``4100000000000019``, so it removes the launch blocker. I can adjust the error handler later.

Reply:
Please let us know if you'd like us to adjust this.